src/analysis/mesh/qgsmeshvirtualdatasetgroup.h

```diff
@@ -0,0 +1,85 @@
+/***************************************************************************
+                         qgsmeshvirtualdatasetgroup.h
+                         ---------------------
+    begin                : June 2020
+    copyright            : (C) 2020 by Vincent Cloarec
+    email                : vcloarec at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSMESHVIRTUALDATASETGROUP_H
+#define QGSMESHVIRTUALDATASETGROUP_H
+
+#include "qgsmeshdataset.h"
+#include "qgsmeshcalcnode.h"
+#include "qgsmeshdatagenerator.h"
+
+#include "qgsmeshlayertemporalproperties.h"
+
+#define SIP_NO_FILE
```

Comment:
I think it would be worth to have QgsMeshVirtualDatasetGroup in python api. also i think the QgsMeshVirtualDatasetGroup class should be part of core and not analysis library...

Reply:
the virtual dataset group is dependent of the mesh calculator.

Reply:
ok